app/views/students/show.html.haml

```diff
@@ -32,4 +32,4 @@
   .fr-btns-group.fr-btns-group--inline
     = link_to "Ajouter une PFMP", new_class_student_pfmp_path(@classe, @student), class: 'fr-btn'
     - if @student.current_schooling.attributive_decision.attached?
-      = link_to "Voir la décision d'attribution", url_for(@student.current_schooling.attributive_decision), class: 'fr-btn fr-btn--secondary'
+      = link_to "Voir la décision d'attribution", url_for(@student.current_schooling.attributive_decision), class: 'fr-btn fr-btn--secondary', target: :blank
```

Comment:
pas de target blank please

Reply:
J'en ai parlé avec Melo, et on était tous les deux d'accord pour dire que c'était plus cohérent avec un target blank pour l'ouverture d'un document qui te fait complètement sortir de l'app

Reply:
Pourquoi tu es contre les target blank ?

Reply:
parce que ça casse le flot normal (`target=self`) et ça pose des problèmes d'accessibilité. C.F 

https://dev.to/cerchie/3-ways-to-make-target-blank-accessible-3lan
https://css-tricks.com/use-target_blank/
https://design-system.service.gov.uk/styles/links/

Si on a des retours en COMUT on peut revisiter mais c'est une assumption de power user le target=blank

Reply:
> c'est une assumption de power user 

Probable. Je te suis et je l'enlève.